doc/source/contributing/index.rst

```diff
@@ -42,9 +42,29 @@ Build documentation
 To build the PyFluent documentation locally, run the following commands in the root
 (``pyfluent``) directory of the repository:
 
+Windows
+~~~~~~~
+
+Download `poppler zip file <https://github.com/oschwartz10612/poppler-windows/releases/tag/v24.08.0-0>`_ and add the bin directory to your PATH.
```

Comment:
@hpohekar A few recommendations here.
1. There are a few files located at the url. We should specify Release-24.08.0-0.zip.
2. We should specify the unzip step between the steps of downloading and adding to the path.
3. We should lay the whole thing out more clearly:
```
1. Install poppler
     i. download specific file
     ii. unzip
     iii. update path
2. Execute the following commands
```